pnpm-lock.yaml

```diff
@@ -0,0 +1,4047 @@
+lockfileVersion: '6.0'
```

Comment:
@huijing was there a reason for not committing this originally?

Reply:
this forces people to use pnpm but if they wanted to use some other package manager they would run into problems. are we mandating that everyone use pnpm for everything?

Reply:
> this forces people to use pnpm

Only for the developers of this plugin. We can remove the fill & add it to .gitignore if you prefer using something else